desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml

```diff
@@ -44,10 +60,15 @@
             This application lets you run Flash content on your computer without a browser in-between,
             and take full advantage of your GPU and system resources to get those extra frames when playing intense games.
         </p>
+        <p xml:lang="ar">يتيح لك هذا التطبيق تشغيل محتوى الفلاش على كمبيوترك دون الحاجة إلى متصفح بينها، والإستفادة الكاملة من وحدة معالجة الرسومات وموارد النظام للحصول على إطارات إضافية عند ممارسة الألعاب.</p>
```

Comment:
Why is there a non-RTL punctuation mark? :thinking:

Reply:
Never mind, it's because some punctuation characters are bidirectional, I guess they just render this way because they assume LTR from their towards-end neighbors and not RTL from their towards-start neighbors.